src/lib/http.coffee

```diff
@@ -25,6 +25,13 @@ module.exports = (dnschain) ->
             @rateLimiting = gConf.get 'rateLimiting:http'
             app = express()
 
+            if gConf.get('http:cors')
+              @log.warn "Enabling CORS HTTP headers"
+              app.use (req, res, next) =>
+                res.header "Access-Control-Allow-Origin", "*"
+                res.header "Access-Control-Allow-Headers", "Origin, X-Requested-With, Content-Type, Accept"
+                next()
```

Comment:
Have you tested that this works to your satisfaction? (It looks right to me, but I have no interest in this feature and so no plans to test that the headers get added).